klippy/stepper.py

```diff
@@ -124,6 +124,14 @@ def get_tag_position(self):
         return self._tag_position
     def set_tag_position(self, position):
         self._tag_position = position
+    def get_past_commanded_position(self, clock):
+        ffi_main, ffi_lib = chelper.get_ffi()
+        sq = self._stepqueue
+        mcu_pos = ffi_lib.stepcompress_find_past_position(sq, clock)
+        mcu_pos_dist = mcu_pos * self._step_dist
+        if self._invert_dir:
```

Comment:
Hi @KevinOConnor, I am testing this on my fork now and am getting an increase in the negative direction with positive extrusion.  I have self._invert_dir = True in my printer.cfg, so maybe inversion is already taken care of in stepcompress_find_past_position. Please confirm, thanks.